test/feature/use_middleware_helper.feature

```diff
@@ -38,6 +38,38 @@ Feature: BaseController useMiddleware helper
       | GET     | /users/1 | foo           |
 
 
+  Scenario Outline: specify multiple middleware
+    Given a file "controllers/users_controller.coffee" with the content
+      """
+      {BaseController} = require 'exprestive'
+
+      middle1 = (req, res, next) ->
+        req.custom1 = 'foo'
```

Comment:
Currently following the example of this [feature](https://github.com/alexdavid/exprestive/blob/b246bcff2bb913ad37fd8b0c9aaf0fba82840d2e/test/feature/middleware_support.feature#L3). Can deal with making this better in another PR.